ee/query-service/app/api/license.go

```diff
@@ -64,7 +64,7 @@ func (ah *APIHandler) applyLicense(w http.ResponseWriter, r *http.Request) {
 		RespondError(w, model.BadRequest(fmt.Errorf("license key is required")), nil)
 		return
 	}
-
+	ctx := auth.AttachJwtToContext(r.Context(), r)
```

Comment:
Why is this done here?

Reply:
It's used [here](https://github.com/SigNoz/signoz/pull/3972/files#diff-571d493907b4315fc1ac269efcf7668db417a5a591c6cd62e582046ed24b1043R263)